core-galleon-pack/src/main/resources/feature_groups/host-server-configs.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="host-server-configs" xmlns="urn:jboss:galleon:feature-group:1.0">
```

Comment:
I don't think this group should exist, or perhaps if it is important to reuse these settings elsewhere it should be called host-example-server-configs. Declaring these three server configs has always just been a kind of example and their specific settings are not meaningful beyond that.

The host.xml file we ship is like that in general.

Reply:
If you don't mind, I'd still keep it. For the purpose of having a dedicated building block that represents the server configs. I get your point and find it reasonable. But I don't feel good about renaming to an "example" block. Especially with how I see the provisioning groups evolving.

Reply:
How would a user use the building block? How much manipulation would they need in order to go from the building block to a useful config?

A building block makes sense, but I'm doubtful about whether the content here is a good one.